target/linux/ath79/dts/ar7240_tplink_tl-wr941nd-v4.dts

```diff
@@ -4,6 +4,6 @@
 #include "ar7240_tplink_tl-wr74xn-v1.dtsi"
 
 / {
-	model = "TP-Link TL-WR941N/ND v4";
-	compatible = "tplink,tl-wr941-v4", "qca,ar7240";
+	model = "TP-Link TL-WR941ND v4";
+	compatible = "tplink,tl-wr941nd-v4", "qca,ar7240";
```

Comment:
When changing the compatible-string of the root node, the old string has to be added to the support-list for the image generation (19.07 got branched, so we will have to take care of sysupgrade compatibility).

Reply:
AFAIK 19.07 is not going to have images for ath79, still ar71xx, so I don't think, that this changes are needed.

Reply:
Having thought about this again, I'd like to propose a different approach:
I personally consider this change a _fix_ (Name is "wrong" to some extent).
Thus, I think it would be best to also have it backported to 19.07.
Since that one hasn't seen even RC tag so far, this would give us the chance to (almost) have it correct from the beginning.

We had a similar discussion already for the WDR4300 to WDR4300 v1 rename, which eventually also had been backported on my request:
master: https://github.com/openwrt/openwrt/commit/ab02584685a5db880c0abe5a51d0213551e8728f
19.07: https://github.com/openwrt/openwrt/commit/96cc390d8808a79f920b6b3fd256cfeb6fd501b2

Since this patch is not connected to patch 1 here anyway, I will make it a separate PR, so we do not depend on 940N testing.
I will also provide a backport PR then.
I think this would be the cleanest solution after all.

Reply:
This discussion is obsolete, as 981213 reported about Chinese 941N below.
So I will just kill this patch.

Reply:
@ynezz Wasn't it the other way around? Drop ar71xx and ship ath79 instead.

The release snapshots would match this: http://downloads.openwrt.org/releases/19.07-SNAPSHOT/targets/

Reply:
I thought 19.07 should ship both targets, because there are some devices in ath79 which are missing in ar71xx and the other way round.

Reply:
@blocktrron ath79 has images generated in 19.07 just because nobody has disabled it yet (make it source only target again in 19.07)